lib/core-client/src/globals/globals.ts

```diff
@@ -1,5 +1,10 @@
-import global from 'global';
+import root from 'window-or-global';
+import { AugmentedGlobal } from '../types';
 
-const { window: globalWindow } = global;
+const rootProxy = root as AugmentedGlobal;
 
-globalWindow.STORYBOOK_REACT_CLASSES = {};
+rootProxy.__STORYBOOK_CLIENT_API__ = undefined;
+rootProxy.__STORYBOOK_STORY_STORE__ = undefined;
+rootProxy.__STORYBOOK_ADDONS_CHANNEL__ = undefined;
+rootProxy.STORYBOOK_REACT_CLASSES = {};
+rootProxy.FEATURES = { previewCsfV3: false };
```

Comment:
what about the other features?

Reply:
it's being added as I untangle and run into them 😄 

Will need some help going over these also to make sure we get it all - and have a talk about how to best route them around, core-client is not a good place